FAQs.md

```diff
@@ -0,0 +1,160 @@
+Bop music player is designed with all the basic features of a music player. Aside from that, it has some other cool features like; 
+
+- Playlists.
+- Equalizer.
+- Grouped menus. 
+- File browser.
+- UI customizations.
+
+---
+
+3. **How do I create a playlist?**
+
+To create a playlist on Bop Music Player;
+
+- navigate to the songs page. To do that; click on the three horizontal bar to access menu options, click the musical note icon which is used to depict "All songs". 
+- On the songs page, mark all the songs you wish to add to a playlist.
+- After marking them, clicking the add to playlist button, which is the first icon at the top right of the app.
+-  If you have already existing playlists, you can decide to add the new songs to it. If not, you can create a new one by typing in the name of the playlist and clicking on create. 
+
+---
+
+4. **Can I edit the playlist?**
+
+At the moment, you can only edit the playlist name. But, the ability to edit the content of the playlist itself is not yet implemented in the latest version of the app. That feature is under development. 
+
+---
+
+5. **Can I delete unwanted playlists?**
+
+Yes, you can delete an unwanted playlist. To delete a playlist;
+
+- Access the playlist page from the options menu.
+- On the playlist page, click the drop-down menu button on the playlist you wish to delete. 
+- Click on the bin icon which signifies to delete.
+
+---
+
+6. **Can I import and export playlists on Bop music player?**
+
+No, playlists can neither be imported nor exported.
+
+---
+
+7. **Can I sort tracks on Bop music player?**
+
+No, you can't sort tracks on the app at the moment. But, this feature will be implemented soon. 
+
+---
+
+8. **Where can I find the Bop music player's equalizer?**
+
+To find the app's equalizer;
+
+- Play any song.
+- When the now playing page opens;
+- Click on the music icon at the right part of the app.
+- After clicking the music icon, another page opens up. At the top right of that page, you will see the equalizer button.
+- Click on the button and the equalizer will open.
+
+---
+
+9. **Can I include/exclude folders from the music library?** 
+
+No, you cannot exclude folders that contain music files from showing in the music library. But, you can decide to play music from a particular folder. 
+
+---
+
+10. **How can I play music from a particular folder?**
+
+To play music from a particular folder;
+
+- Access the file manager on the app. Access the app's options menu.
+- Click on the folder icon
+- Browse folder where you stored the music file(s) you wish to play.
+- Click play all to play all the songs in the folder.
+
+---
+
+11. **Can I customize equalizer settings?**
+
+Yes, you can customize/tune the equalizer to suit it to your preferred sound. 
+
+---
+
+12. **Which file types are supported?**
+
+Bop music player uses the native media player that comes with your Android phone, so as long as a file type is supported by your phone, it's supported by Retro Music.
```

Comment:
Is retro music a kind of derivative to Bop, or you mistakenly wrote another name?

Reply:
Sorry. It was a mistake.